src/installer/installer.go

```diff
@@ -95,8 +95,8 @@ func (i *installer) InstallNode() error {
 	i.Config.Device = i.ops.EvaluateDiskSymlink(i.Config.Device)
 	err := i.cleanupInstallDevice()
 	if err != nil {
+		// Do not change the phrasing of this error message, as we rely on it in a triage signature
 		i.log.Errorf("failed to prepare install device %s, err %s", i.Device, err)
-		return err
```

Comment:
Lets emit an event here too - self-hosted users don't get the benefits of triage signatures so it would be useful too (thanks @eranco74 for the suggestion)

Reply:
sure, can someone point me to some code emitting an event? O:)

Reply:
So, emitting an event is tricky.
The assisted installer can't create an event, what it can do is update the installation progress and in response to some of these updates the assisted-service generates events.
So I suggest you add a call to UpdateHostInstallProgress with the same stage and info message saying that the cleanup failed.
Later we can add the code to the service that identifies the string in the progress info and emits an event. (edited)